### PR TITLE
Make sure Heroku runs migrations every deploy

### PR DESCRIPTION
**Pull request by benthorner**
Unfortunately the postdeploy script in app.json only runs when the app is first created. There's apparently no hook for something that runs every time it's deployed, apart from the start command.

### CODE REVIEW

**Comment by benthorner** [comment]
Ah, I didn't expect they would solve it there. Let's try it.

File: Procfile [hunk 1]
```diff
@@ -1,2 +1,3 @@
+release: rake db:migrate
```
Comment (andrewgarner) [inline Procfile]: I think we should use the bin stub to be consistent with the other processes. Also `bin/rails` is the [new default for running commands](http://guides.rubyonrails.org/command_line.html#bin-rails).

```
release: bin/rails db:migrate
```

Comment (benthorner) [reply Procfile]: I was being consistent with app.json, but OK. I wonder how long this 'default' will take to become the default ;-)

Comment (andrewgarner) [reply Procfile]: I'm fighting the good fight ;-)

**Pull request merged**